electron/src/runtime/lifecycle.ts

```diff
@@ -69,15 +69,17 @@ export const getWebViewId = (contents: WebContents): string | undefined => {
 // Using exit instead of quit for the time being
 // see: https://github.com/electron/electron/issues/8862#issuecomment-294303518
 export const quit = (): void => {
-  logger.info('Initiating app quit ...');
+  logger.info('Initiating app quit...');
   settings.persistToFile();
+  logger.info('Clear cache...');
+  session.defaultSession.clearCache().catch(() => {});
```

Comment:
that's the actual change!

Reply:
Ok, then let's keep this PR to that change and put back the spaces to "...". These were set on purpose.

Reply:
From the copy point of view we should never have spaces before `,.?! etc`.

Shall we fix that in separate PR?

Reply:
No. As I wrote. The spaces were set on purpose. They should be kept.